C/CGAL/build_tarballs.jl

```diff
@@ -49,8 +50,8 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("boost_jll"),
-    Dependency("GMP_jll", v"6.1.2"),
-    Dependency("MPFR_jll", v"4.0.2"),
+    Dependency("GMP_jll"),
+    Dependency("MPFR_jll"),
```

Comment:
Ok, this change should be reverted, otherwise the package won't work in Julia v1.5-

Reply:
Indeed

Reply:
Reverted, thank you :)